university student project edits (#4837)
RasaHQ/rasa: README.md

- Alexa Skills
- Google Home Actions

> Rasa's primary purpose is to help build contextual, and layered

Reviewer: ```suggestion
Rasa's primary purpose is to help build contextual, layered
```

conversations with lots of back-and-forth. To have a real conversation,

> you need to use your memory and build on things said previously.

Reviewer: ```suggestion
your bot needs to have some memory and build on things said previously.
```

Rasa lets you do that in a scalable way.

> There is more background information here:

Reviewer: ```suggestion
Find more background information in this
```

## Where to get help

There is extensive documentation in the [Rasa Docs](https://rasa.com/docs/rasa).

> Make sure to select the correct version to use the correct corresponding documentation.

Reviewer: ```suggestion
Make sure to select the version you installed to see the correct corresponding documentation.
```

### How to contribute
We are very happy to receive and merge your contributions. You can

> find more information about contributing to Rasa (in lots of

Reviewer: ```suggestion
find more information about how to contribute to Rasa (in lots of
```
